app/graphql/mutations/review_create.rb

```diff
@@ -16,7 +16,9 @@ class ReviewMutation < BaseMutation
     argument :safety_meter, Integer, required: true
     argument :gender, String, required: false
 
-    type Types::ReviewType
+    field :review, Types::ReviewType, null: false
```

Comment:
When returning one type leaving it as type is fine. When returning multiple types, the field information is needed to specify the return information.